dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryMisconfigTest.java

```diff
@@ -123,54 +131,68 @@ public static void setUp() throws Exception {
     }
 
     @Test
-    @ExpectedFFDC({ "org.jboss.resteasy.spi.UnhandledException" })
+    @ExpectedFFDC({ "io.opentelemetry.sdk.autoconfigure.spi.ConfigurationException" })
     public void testInvalidExporter() throws Exception {
         server.setMarkToEndOfLog();
 
         new HttpRequest(server, "/" + INVALID_EXPORTER_APP_NAME + "/misconfig/jaxrsclient")
-                        .expectCode(500)
+                        .expectCode(200)
                         .run(String.class);
```

Comment:
Why are these changes needed?

Reply:
I guess the `expectCode` change could be a result of #25626, but if that was the case, I'd have expected this test to fail in the PB for that build.

Reply:
We checked and these tests _are_ failing in the build since #25626 was merged. The right tests were not requested for the personal build on that PR.